pallets/ah-migrator/src/vesting.rs

```diff
@@ -86,3 +89,64 @@ pub mod alias {
 		V1,
 	}
 }
+
+#[cfg(feature = "std")]
+impl<T: Config> crate::types::AhMigrationCheck for VestingMigrator<T> {
+	type RcPrePayload =
+		Vec<(Vec<u8>, Vec<BalanceOf<T>>, Vec<GenericVestingInfo<BlockNumberFor<T>, BalanceOf<T>>>)>;
+
+	type AhPrePayload =
+		Vec<(Vec<u8>, Vec<BalanceOf<T>>, Vec<GenericVestingInfo<BlockNumberFor<T>, BalanceOf<T>>>)>;
+
+	fn pre_check(_: Self::RcPrePayload) -> Self::AhPrePayload {
+		pallet_vesting::Vesting::<T>::iter()
```

Comment:
Actually there should be no vesting schedules on the asset hub before the migration starts.  
Can you please check that it is empty?

Reply:
Thanks, fixed